diagnostics-app/karma.conf.js

```diff
@@ -0,0 +1,44 @@
+// Karma configuration file, see link for more information
```

Comment:
can we move test-related files into a separate folder. e.g. diagnostics-app/test/

Reply:
Thanks for sharing! Let's keep them in the same directory, then.

Also, let's not forget to add a test/ folder when it comes to integration tests, according to this: https://angular.io/guide/testing#place-your-spec-files-in-a-test-folder